open_instruct/vllm_utils3.py

```diff
@@ -54,6 +51,19 @@
 
 logger = logger_utils.setup_logger(__name__)
 
+DEFAULT_WORKERS = 2
+TOOL_WORKERS = 20
+INIT_TIMEOUT_S = 120
+WEIGHT_UPDATE_TIMEOUT_S = 120
+DRAIN_TIMEOUT_S = 300.0
+SHOULD_STOP_CACHE_TIMEOUT_S = 5
+PREFETCH_SLEEP_S = 1
+QUEUE_GET_TIMEOUT_S = 0.1
+COMPLETION_QUEUE_TIMEOUT_S = 1.0
+PROCESS_SLEEP_S = 0.1
+RAY_WAIT_TIMEOUT_S = 0.1
+KV_CACHE_RETRIES = 5
+KV_CACHE_RETRY_SLEEP_S = 0.2
 WEIGHT_UPDATE_SLEEP_INTERVAL_S = 0.1
```

Comment:
Could we add short explanations of each constant? Some are obvious (weight_update_timeout), but others are a bit less clear (default_workers, kv_cache_retries)

Reply:
I went over all of these. A bunch of them were redundant and could be removed; others, imo, should be handled by timeouts in `grpo_fast.py` on the actual `ray.get` calls, so I removed them. I think the remaining ones are clear from the names. What do you think?